tests/AutoReview/GAE2ETest.php

```diff
@@ -31,7 +31,7 @@ class GAE2ETest extends TestCase
 
     public function test_github_actions_executes_all_the_e2e_tests(): void
     {
-        $expected = array_diff(E2ECollector::getE2ENames(), self::IGNORED_E2E_TESTS);
+        $expected = array_values(array_diff(E2ECollector::getE2ENames(), self::IGNORED_E2E_TESTS));
```

Comment:
I don't understand this change: isn't the `assertEqualsCanonicalizing()` supposed to not care about the keys difference?